tests/test_views.py

```diff
@@ -33,7 +33,7 @@ def test_basic_view(schema, schema_map, expected_result):
     t1 = Table('t1', sa.MetaData(schema=schema),
                sa.Column('col1', sa.Integer(), primary_key=True),
                sa.Column('col2', sa.Integer()))
-    selectable = sa.sql.select([t1])
+    selectable = sa.sql.select(t1)
```

Comment:
Why remove the wrapping list? Was this causing errors in 2.0?

Reply:
Yes previous version expected a list of entities as first select argument. In the latest version we only provide the list as method arguments cf https://docs.sqlalchemy.org/en/20/core/selectable.html#sqlalchemy.sql.expression.select